src/Microsoft.AspNet.Diagnostics.Elm/Views/LogPage.cs

```diff
@@ -306,6 +306,8 @@ public LogPage()
         #pragma warning disable 1998
         public override async Task ExecuteAsync()
         {
+            Response.ContentType = "text/html; charset=utf-8";
+            Response.ContentLength = null; // Clear any prior Content-Length
```

Comment:
Those are calling _next and then catching exceptions. This middleware does not call _next if it finds a path match.